### DISCUSSION TITLE
Field Day 2024

### DISCUSSION THREAD

**Discussion by kratcliff** [mbridak/FieldDayLogger#43] [General]
I'm going to try this out for Field Day this weekend! Probably just standalone on my laptop since I'm getting such a late start. But I love the idea of syncing to a pi over the local WiFi!

Anyone else going to use it for 2024?

**Comment by mbridak** [comment]
Excellent. Hope it works out for you. I unfortunately will not be able to
operate this Field Day. First miss since getting my ticket.

**Reply by kratcliff** [reply]
I just learned that one club member plans to use N1MM. I run linux on my laptop, so I'd have to use WINE if I wanted to run N1MM. But what can you tell me about the N1MM status packets in FieldDayLogger? What functionality would that give me if I was on the same WiFi network as an N1MM user?

**Comment by mbridak** [comment]
Hi. I could not tell you much as to how n1mm will react to the packets. I
mainly included them so you could be tracked on AA0Z's nodered dashboard.
Now that I'm retired and the kids are in summer vacation, I can devote more
time to development. Most of my development time recently has been focused
on not1mm. But anything good will trickle down.
